tests/braille-specs/ancient-languages-de.yaml

```diff
@@ -0,0 +1,23 @@
+
+# Yaml Test For Biblical  Language/Ancient Near Eastern Studies German system
```

Comment:
As this new table does not represent a single braille code, but is merely an aggregation of a bunch of related tables, it doesn't make that much sense to have a YAML file for it IMO. More appropriate would be to test this table against all the test files for the included tables. It would not only be more logical, it would also result in a more complete test suite. Any test that is added to any of the other test files will automatically be run on his aggregator table.

The only reason for having a special "ancient-languages" test file is to show how different languages could be mixed, e.g. in a single sentence.

Reply:
Done